spec/requests/citizens/check_answers_spec.rb

```diff
@@ -127,9 +130,10 @@
 
     context 'applicant does not have any capital restrictions' do
       let(:legal_aid_application) { create :legal_aid_application, :with_everything }
-      let!(:restriction) { nil }
+      let(:has_restrictions) { false }
+      let!(:restrictions_details) { '' }
```

Comment:
because `restrictions_details` is an attribute of `legal_aid_application` and not another table, you don't need the `!`.

https://relishapp.com/rspec/rspec-core/v/2-5/docs/helper-methods/let-and-let
*let is lazy-evaluated: it is not evaluated until the first time the method it defines is invoked. You can use let! to force the method's invocation before each example.*

And this line is not needed either:
`let(:legal_aid_application) { create :legal_aid_application, :with_everything }`.
We can use the same `legal_aid_application` from line 7